hadoop-tools/hadoop-azure/src/site/markdown/abfs.md

```diff
@@ -888,38 +888,6 @@ specified SSL channel mode. Value should be of the enum
 DelegatingSSLSocketFactory.SSLChannelMode. The default value will be
 DelegatingSSLSocketFactory.SSLChannelMode.Default.
 
-### <a name="encryptionconfigoptions"></a> Encryption Options
```

Comment:
Removed the documentation added for 'Customer-Provided Global Key', 'Encryption Context Provider'.